filebeat/tests/system/test_modules.py

```diff
@@ -170,6 +170,7 @@ def _test_expected_events(self, test_file, objects):
 
         for ev in expected:
             found = False
+            clean_keys(ev)
```

Comment:
Should we wait till https://github.com/elastic/beats/pull/9852 is merged and then rebase this PR on `6.x`?